build/rpms/10-kubeadm.conf

```diff
@@ -1,7 +1,5 @@
 # Note: This dropin only works with kubeadm and kubelet v1.11+
 [Service]
-Environment="KUBELET_KUBECONFIG_ARGS=--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf"
```

Comment:
How are you updating the systemd kubelet start now without these args?

Reply:
During the `init` or `join` phase when the kubelet's arguments are written to `/var/lib/kubelet/kubeadm-flags.env`, I am additionally writing these flags to that environment file. Then when the restart is issued the flags are picked up.

Reply:
please see: https://github.com/kubernetes/kubernetes/pull/69985/files#diff-1a4e0bdeb8495d00560f3e20354abf41R80